validation-example.conf

```diff
@@ -44,12 +44,12 @@ pipelines: [
     stages: [
       {
         class = "com.kmwllc.lucille.stage.Concatenate"
-        format_string = "{city}, {state}, {country}"
+        formatString = "{city}, {state}, {country}"
       },
       {
         class = "com.kmwllc.lucille.stage.Concatenate"
         dest = "dest"
-        format_string = "{city}, {state}, {country}"
+        formatString = "{city}, {state}, {country}"
         default_inputs3 {
```

Comment:
default_inputs3 still has an underscore

Reply:
updated in [63261d5](https://github.com/kmwtechnology/lucille/pull/203/commits/63261d5ae1fc5ee23031a4e9f3c14d620c7e13d8)